company-profiles/you-are-launched.md

```diff
@@ -0,0 +1,25 @@
+# You are launched
+
+## Company blurb
+
+You are launched (urlaunched.com) - is a top-rated Ukrainian custom app development partner who knows what to do with ideas, MVPs, and how to launch startups in just 3 months.
+
+## Company size
+
+24 people in 4 countries (Aug 2022)
```

Comment:
4 countries mentioned here but **Region** is mentioned as only Ukraine and Poland. The Region is for where the company hires within, are you only hiring in Ukraine and Poland right now?

Reply:
Hello.
We have team members from Ukraine, Poland, Germany, and Spanish.
But at the moment we have opened positions in Ukraine and Poland only.

Thanks